docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -49,20 +49,95 @@ refs:
+
+The supported location formats are:
+
+- Latitude and longitude
+- Geohash
+- Lookup codes: country, US states, or airports
+
+To learn more, refer to [Location mode](#location-mode).
+
+Geomaps also support additional fields with various data types to define things like labels, numbers, heat sizes, and colors.
+
+### Example - Latitude and longitude
+
+If you plan to use latitude and longitude coordinates, the dataset must include at least two fields (or columns): one called `latitude` (you can also use`lat`), and one called `longitude` (also `lon` or `lng`). When you use this naming convention, the visualization automatically detects the fields and displays the elements. The order of the fields doesn't matter as long as there is one latitude and one longitude.
+
+| Name            | latitude  | longitude | value |
+| --------------- | --------- | --------- | ----- |
+| Disneyland      | 33.8121   | -117.9190 | 4     |
+| DisneyWorld     | 28.3772   | -81.5707  | 10    |
+| EuroDisney      | 48.867374 | 2.784018  | 3     |
+| Tokyo Disney    | 35.6329   | 139.8804  | 70    |
+| Shanghai Disney | 31.1414   | 121.6682  | 1     |
+
+If your latitude and longitude fields are named differently, you can specify them, as indicated in the [Location mode](#location-mode) section.
+
+### Example - Geohash
+
+If your location data is in geohash format, the visualization requires at least one field (or column) containing location data.
+
+If the field is named `geohash`, the visualization automatically detects the location and displays the elements. The order of the fields doesn't matter and the data set can have multiple other numeric, text, and time fields.
+
+| Name      | geohash      | trips |
+| --------- | ------------ | ----- |
+| Cancun    | d5f21        | 8     |
+| Honolulu  | 87z9ps       | 0     |
+| Palm Cove | rhzxudynb014 | 1     |
+| Mykonos   | swdj02ey9gyx | 3     |
+
+If your field containing geohash location data is not named as above, you can configure the visualization to use geohash and specify which field to use, as explained in the [Location mode](#location-mode) section.
+
+### Example - Lookup codes
+
+The geomap visualization can identify locations based on country, airport, or US state codes.
+
+For this configuration, the dataset must contain at least one field (or column) containing the location code.
+
+If the field is named `lookup`, the visualization automatically detects it and displays points based on country codes.
+
+| Year | lookup | gdp       |
+| ---- | ------ | --------- |
+| 2016 | MEX    | 104171935 |
+| 2016 | DEU    | 94393454  |
+| 2016 | FRA    | 83654250  |
+| 2016 | BRA    | 80921527  |
+| 2016 | CAN    | 79699762  |
+
+The other location types&mdash; airport codes or US state codes&mdash;aren't automatically detected.
+
+If you want to use other codes or give the field a custom name, you can follow the steps in the [Location mode](#location-mode) section.
+
```

Comment:
These are great examples! Is the intention to have these at the top and have the users dig deeper down below if needed?

Reply:
@srperf FTW! Definitely intended to be down here.